### ISSUE TITLE
Empty MetaData Values

### ISSUE THREAD

**Issue by philipbaileynar** [Riverscapes/riverscapes-tools#1039]
Hydro Context contains several metadata keys that have no values. Technically the spec allows for this, but it is currently breaking some of our systems such as WebViewer. @MattReimer  is patching these, but it's a good idea to avoid keys with no values.

```xml
<Meta name="SourceUrl" ext="dataset" type="url"/>
<Meta name="DataProductVersion" ext="dataset"/>
```

@jtgilbert can you please fix Hydro Context and either provide values for all MetaData or remove the entries from the `project.rs.xml`?

<img width="794" alt="Screenshot 2024-10-24 at 3 00 39 PM" src="https://github.com/user-attachments/assets/fb46a5f4-bde3-4ae9-82bf-9213ff697e0a">

**Comment by jtgilbert** [comment]
@philipbaileynar, this isn't an easy fix....
the lyr_discriptions.json present in every tool has three elements associated with each layer: a description, a source url, and a product version. This was mainly to keep track of versions and sources for layers in RSContext. So, b/c the source url and version aren't relevant for most layers, they're left blank (empty quotes) in lyr_description.json. The augment metadata function then adds all this info, creating an empty tag if there's not info filled in. 
Bottom line, almost every project in the exchange probably has some of these empty meta tags.

**Comment by philipbaileynar** [comment]
OK, maybe that's fine. @MattReimer ?

**Comment by MattReimer** [comment]
Empty metadata is semantically fine as long as it's intended
